### ISSUE TITLE
Skip auth by net-rule

### ISSUE THREAD

**Issue by CosmoFox** [IceWhaleTech/CasaOS#787]
qBittorrent have a nice feature for web admin panel - skip(bypass) authentication if you connecting from localhost or some lans. I think this will be awesome feature for lovely CasaOS.
![image](https://user-images.githubusercontent.com/9054349/209551415-3805ea5f-69b2-4e0f-b8d8-f188117e30f1.png)

**Comment by JohnGuan** [comment]
@CosmoFox 
That means you want CasaOS to have a similar feature that allows users on a LAN or a specific network segment to use it login-free, right?

**Comment by CosmoFox** [comment]
Absolutely. For example: in the next room I have a touch monitor without a keyboard, on which CasaOS is always open. I want to monitor apps and forget about entering my password..
